src/styles/global.css

```
@@ -0,0 +1,267 @@
+   ANIMATION UTILITIES
+   ═══════════════════════════════════════════════════════════════════════════ */
+
+.ease-out-expo {
+  transition-timing-function: var(--ease-out-expo);
+}
+
+.ease-out-quint {
+  transition-timing-function: var(--ease-out-quint);
+}
+
+@keyframes dotScroll {
```

Stylelint (17.2.0)

[error] Expected keyframe name "dotScroll" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)